Standards/scs-0210-v2-k8s-version-policy.md

```diff
@@ -0,0 +1,81 @@
+---
+title: SCS K8S Version Policy
+type: Standard
+status: Draft
+stabilized_at: 2023-08-16
+track: KaaS
+---
+
+## Introduction
+
+The Kubernetes project maintains multiple release versions including their patched versions.
+In the project, the three most recent minor releases are actively maintained, with a fourth
+version being in development. As soon as a new minor version is officially released,
+the oldest version is dropped out of the support period.
+Kubernetes supports its releases for around 14 months. 12 of these are the standard
+support period. The remaining 2 months are end-of-life support for things like:
+
+- CVEs (under the advisement of the Security Response Committee)
+- dependency issues (including base image updates)
+- critical core component issues
+
+More information can be found under [Kubernetes Support Period].
+
+The Kubernetes releases are planned for about 4 months, which usually results in about
+**3 minor** releases per year [Kubernetes Release Cycle].
```

Comment:
d
```suggestion
**3 minor** releases per year [Kubernetes Release Cycle](https://kubernetes.io/releases/release/).
```

Reply:
I just recognized, that links are listed at the end of the document. Is this a common practice in SCS standards?

Reply:
No. Linking docs that explain a statement or term is best be done in place. A link I would click if I need an explanation or definition or want to see the source of truth for a statement.

A references section at the end is more for further reading or background reading (in case I'm new to the topic).
Also link to related content, e.g. related standards.

Reply:
Should we really put links in different locations depending on their relation to the standard? I would rather try to put everything in the final section and check if the link nevertheless works in the document itself.

Reply:
In favor of readability, I would add links in place. And put, as @garloff mentioned, additional documents, not related to a special term, at the bottom of the document.

Reply:
Put the links in the text.